lessons/03 - Customizing Your React Leaflet Map with a Mapbox Basemap Style/README.md

```diff
@@ -82,7 +82,13 @@ We'll update our Map compopnent that we set up in the previous lesson to use our
 
 ### 1. Extra Env Vars
 
-TODO
+Similar to [exercise 4](lessons/03%20-%20Customizing%20Your%20React%20Leaflet%20Map%20with%20a%20Mapbox%20Basemap%20Style#4-create-an-environment-variable-for-the-api-key), we can pull out other sensitive account data out of our code and into environment variables.
+
+Put your User ID and Style ID into env variables and call them within your application.
+
+Where We'll Make Changes
+ - lessons/env.shared
+ - ./src/App.js
```

Comment:
Exercise 4 didn't reference App.js so maybe we remove this line?

Reply:
i think having App.js makes sense - because i think they would have to update the variables in the app to use them, where i have the other ones already set up to be used. i think it would make sense to add App.js in the other one too, think i just missed adding it there. ill update that one